docs/Images.md

```diff
@@ -1,34 +1,26 @@
 # Required Docker Images
 
-The Splunk operator requires three docker images to be present or available
-to your Kubernetes cluster:
+The Splunk Operator requires three docker images to be present or available to your Kubernetes cluster:
 
 * `splunk/splunk-operator`: The Splunk Operator image (built by this repository)
 * `splunk/splunk:8.1.0`: The [Splunk Enterprise image](https://github.com/splunk/docker-splunk) (8.1.0 or later)
 * `splunk/spark`: The [Splunk Spark image](https://github.com/splunk/docker-spark) (used when DFS is enabled)
 
-All of these images are publicly available on [Docker Hub](https://hub.docker.com/).
-If your cluster does not have access to pull from Docker Hub, you will need to
-manually download and push these images to an accessible registry. You will
-also need to specify the location of these images by using either an environment
-variable passed to the operator or adding additional `spec` parameters to your 
+All of these images are publicly available, and published on [Docker Hub](https://hub.docker.com/).
+
+If your cluster does not have access to pull directly from Docker Hub, you will need to manually download and push these images to an accessible registry. You will also need to specify the location of these images by using an environment variable passed to the operator, or by adding additional `spec` parameters to your 
 custom resource definition.
 
-Use the `RELATED_IMAGE_SPLUNK_ENTERPRISE` environment variable or the `image`
-custom resource parameter to change the location of the Splunk Enterprise
-image. Use the `RELATED_IMAGE_SPLUNK_SPARK` environment variable or the
-`sparkImage` parameter to change the location of the Spark image. Please see the
-[Advanced Installation Instructions](Install.md) or
-[Custom Resource Guide](CustomResources.md) for more details.
+Use the `RELATED_IMAGE_SPLUNK_ENTERPRISE` environment variable or the `image` custom resource parameter to change the location of your Splunk Enterprise image. Use the `RELATED_IMAGE_SPLUNK_SPARK` environment variable or the`sparkImage` parameter to change the location of the Spark image. 
+
+For additional detail, see the [Advanced Installation Instructions](Install.md) page, and the [Custom Resource Guide](CustomResources.md) page.
 
 
 ## Using a Private Registry
 
-If your Kubernetes workers have access to pull from a Private registry, it is
-easiest to retag and push the required images to directly to the private registry.
+If your Kubernetes workers have access to pull from a Private registry, it is easy to retag and push the required images to directly to the private registry.
```

Comment:
nit: not related to your change specifically, but we have inconsistent capitalization between "Private registry" and "private registry"